learning_assistant/toggles.py

```diff
@@ -37,15 +36,15 @@ def _is_learning_assistant_waffle_flag_enabled(flag_name, course_key):
         return False
 
 
-def course_content_enabled(course_key):
+def chat_history_enabled(course_key):
     """
-    Return whether the learning_assistant.enable_course_content WaffleFlag is on.
+    Return whether the learning_assistant.enable_chat_history WaffleFlag is on.
     """
-    return _is_learning_assistant_waffle_flag_enabled(ENABLE_COURSE_CONTENT, course_key)
+    return _is_learning_assistant_waffle_flag_enabled(ENABLE_CHAT_HISTORY, course_key)
 
 
-def chat_history_enabled(course_key):
+def v2_endpoint_enabled():
```

Comment:
I did not make this a course waffle flag as I don't believe we need a course by course roll out.